### DISCUSSION TITLE
AspNetCore Server Configuration

### DISCUSSION THREAD

**Discussion by theafricaninternetgroup** [MarimerLLC/csla#3064] [Questions] [answered]
I am trying to port my CSLA.NET 5 app to CSLA.NET 6.0 I am following the example as given in a BlazorExampleSample the sample uses
 preview version 6.0.0-R21110203 however the following seem not to compile in version 6.0 or there is something I am missing
```
builder.Services.AddCsla(options => options
  .DataPortal()
    .AddServerSideDataPortal()
    .UseLocalProxy());
```

**Comment by rockfordlhotka** [comment, marked as answer]
Are you sure you are looking at the current `main` branch? I see the `BlazorExample` project referencing v6.0.

Oh, it looks like the `BlazorServerExample` solution has projects references v6.0 and v6.1 - at least on my local machine. That won't work, because they all need to be targeting the same version of CSLA. I updated everything to v6.1 and it builds fine.
